truetype/truetype.go

```diff
@@ -95,11 +99,16 @@ type cm struct {
 	start, end, delta, offset uint32
 }
 
+// An nr holds a font's name, style
+type nr struct {
```

Comment:
Names in the global scope generally need to be longer than e.g. struct fields or local variables. The latter have lot more context where they're used. Instead:

type nameRecord struct { etc }